### ISSUE TITLE
n <  1 not converting to MathML correctly

### ISSUE THREAD

**Issue by opoudjis** [plurimath/plurimath#103]
Take the expression `lxm_input = "n < 1"`, which is legal AsciiMath, and (I think) legal LaTeX.

We expect `n < 1` to map to the MathML:

````
               <math xmlns="http://www.w3.org/1998/Math/MathML">
                 <mi>n</mi>
                 <mo>&lt;</mo>
                 <mn>1</mn>
               </math>
````

And indeed, the existing asciimath and latexmath converters were generating that (with the latter inserting the elements within `mrow`

Plurimath is parsing the expressions correctly:

````
#<Plurimath::Math::Formula:0x000000010458fa48                              
 @value=                                                                   
  [#<Plurimath::Math::Function::Text:0x000000010459c0e0 @parameter_one="n">,
   #<Plurimath::Math::Symbol:0x0000000104596898 @value="<">,               
   #<Plurimath::Math::Number:0x0000000104595038 @value="1">]>  
````

for latex input, and

````
#<Plurimath::Math::Formula:0x000000010452ec70
 @value=
  [#<Plurimath::Math::Symbol:0x000000010442f1f8 @value="n">,
   #<Plurimath::Math::Formula:0x000000010450f780
    @value=[#<Plurimath::Math::Symbol:0x0000000104454ef8 @value="<">, 
                    #<Plurimath::Math::Number:0x0000000104476d28 @value="1">]>]>
````

for asciimath input

But the MathML conversion for both is incorrect. For the latex expression,

`Plurimath::Math.parse("n < 1", "latex").to_mathml`

is yielding

````
<math xmlns='http://www.w3.org/1998/Math/MathML' display='block'>
<mstyle displaystyle='true'>
<mtext>n</mtext><mo><</mo><mn>1</mn>
</mstyle>
</math>
````

the _n_ is not being recognised as an `mi`; the `mo` needs to have its `<` escaped as `&lt;`

And 

`Plurimath::Math.parse("n < 1", "asciimath").to_mathml`

is yielding:

````
<math xmlns='http://www.w3.org/1998/Math/MathML' display='block'>
<mstyle displaystyle='true'>
<mo>n</mo>
<mrow><mo><</mo><mn>1</mn></mrow>
</mstyle>
</math>
````

Now the _n_ is being recognised incorrectly as `mo` not `mi`

**Comment by suleman-uzair** [comment]
@opoudjis 
It seems like you are testing these examples using the previous code/gem version, you should use the gem's main branch or release a new version of the gem for the latest code.

**Comment by ronaldtse** [comment]
@suleman-uzair can you git tag a new version in this repo so it automatically releases a new version? Thanks.

**Comment by opoudjis** [comment]
````
Plurimath::Math.parse("n < 1", "latex")

#<Plurimath::Math::Formula:0x0000000104cec6d8                              
 @value=                                                                   
  [#<Plurimath::Math::Symbol:0x0000000104d0f138 @value="n">,               
   #<Plurimath::Math::Unicode:0x0000000104cffdf0 @value="&#x3c;">,         
   #<Plurimath::Math::Number:0x0000000104cf7628 @value="1">]>  

Plurimath::Math.parse("n < 1", "latex").to_mathml

<math xmlns=\"http://www.w3.org/1998/Math/MathML\" display=\"block\">
<mstyle displaystyle=\"true\">
<mi>n</mi>
<mi>&#x3c;</mi>
<mn>1</mn>
</mstyle>
</math>
````

The < is being encoded correctly as an HTML escape, but we do expect this to be `<mo>` not `<mi>`.

````
Plurimath::Math.parse("n < 1", "asciimath")

#<Plurimath::Math::Formula:0x0000000104d4f4b8                              
 @value=                                                                   
  [#<Plurimath::Math::Symbol:0x0000000104d978f8 @value="n">,
   #<Plurimath::Math::Symbol:0x0000000104d6c900 @value="&#x3c;">,
   #<Plurimath::Math::Number:0x0000000104d64de0 @value="1">]>

Plurimath::Math.parse("n < 1", "asciimath").to_mathml

<math xmlns=\"http://www.w3.org/1998/Math/MathML\" display=\"block\">
<mstyle displaystyle=\"true\">
<mi>n</mi>
<mi>&#x3c;</mi>
<mn>1</mn>
</mstyle>
</math>
````

Same issue.

I know this is a very low level thing to be bringing up, but there is semantics in the MathML that isn't there in the asciimath or latex — the operands are expected to be tagged as such, even though we are outputting Presentation MathML; and the asciidoctor asciimath recognises this as an `mo`. So this issue remains open.

**Comment by suleman-uzair** [comment]
Fixed this issue in #110
